internal/cmd/asyncapi/command_export.go

```diff
@@ -159,7 +150,7 @@ func (c *command) export(cmd *cobra.Command, _ []string) error {
 }
 
 func (c *command) getChannelDetails(details *accountDetails, flags *flags) error {
-	output.Printf("Adding operation: %s\n", details.channelDetails.currentTopic.GetTopicName())
+	output.Printf("Adding topic: %s\n", details.channelDetails.currentTopic.GetTopicName())
```

Comment:
Can we wait until the topic is added, and then print this message in past-tense?